substrate/client/basic-authorship/src/basic_authorship.rs

```diff
@@ -539,19 +543,24 @@ where
 	}
 
 	/// Prints a summary and does telemetry + metrics.
+	///
+	/// - `block`: The block that was build.
+	/// - `end_reason`: Why did we stop producing the block?
+	/// - `block_took`: How long did it took to produce the actual block?
+	/// - `propose_took`: How long did the entire proposing took?
 	fn print_summary(
 		&self,
 		block: &Block,
 		end_reason: EndProposingReason,
 		block_took: time::Duration,
-		propose_with_took: time::Duration,
+		propose_took: time::Duration,
```

Comment:
this could be info!'ed too

Reply:
I mean this is only includes the storage proof extraction, not really something we also need to print. IMO.